heltour/tournament/forms.py

```diff
@@ -344,3 +354,18 @@ def __init__(self, *args, **kwargs):
         reg = kwargs.pop('reg')
         super(MoveLateRegForm, self).__init__(*args, **kwargs)
         self.fields['prev_round'].initial = reg.round.number
+
+class CreateTeamsForm(forms.Form):
```

Comment:
Please add a confirmation field (like [here](https://github.com/cyanfish/heltour/blob/c07d7f698794fd34c699bb43de91fb4181696cfb/heltour/tournament/forms.py#L296)) and make sure to validate it, so that it takes more than a single click from this page
e.g. "Yes, I'm sure - delete %d teams and regenerate"